openmdao/components/interp_util/tests/test_interp_nd.py

```diff
@@ -79,8 +79,9 @@ def test_interp_spline_bsplines(self):
         ycp = np.array([5.0, 12.0, 14.0, 16.0, 21.0, 29.0])
         n = 50
         x = np.linspace(1.0, 12.0, n)
+        x_normalized = x / x[-1]  # normalize x to range 0 to 1
 
-        interp = InterpND(method='bsplines', num_cp=6, x_interp=x)
+        interp = InterpND(method='bsplines', num_cp=6, x_interp=x_normalized)
```

Comment:
Sorry, I should've updated this after the Slack discussion, but I agree. A temporary, but clunky, solution with the current interface would be to set the final value of `x_interp` to what you want the top of the spline x range to be and then use `src_indices` to ignore the final sampled point.

Reply:
I think we want to set both the start and end value of x_cp, with the default (none) meaning they are just on the end points of x_interp. 

BTW, what we have presently is not great, because it assumes x_interp[0] is at 0, and x_interp[-1] is the max magnitude. If we wanted x_interp to go from [5, 12], or [-10, 0], it would also not be quite right. The equation for the correct transformation is the same as our driver scaling (via ref, ref0).